src/lib/hyrise.hpp

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "boost/container/pmr/memory_resource.hpp"
+#include "concurrency/transaction_manager.hpp"
+#include "storage/storage_manager.hpp"
+#include "utils/plugin_manager.hpp"
+#include "utils/singleton.hpp"
+
+namespace opossum {
+
+class Hyrise : public Singleton<Hyrise> {
+ public:
+  // Resets the Hyrise state by deleting the Plugin-, Storage-, and TransactionManager
```

Comment:
"by deleting its members (e.g., StorageManager)". Otherwise, that comment might become outdated when a member is added.